pods-framework/pods
# Disallow group deletion after fields have been moved in or out of the group

## Description

Disallows groups from being deleted until field order has been saved to prevent issue with fields being removed or orphaned. 

## Related GitHub issue(s)

Fixes #6937.

## Testing instructions

1. Drag a field from one group to another.
2. Click "Delete" on either the previous or new group, and see an alert that the Pod must be saved.
3. Try to delete a Group that was unaffected by the Field drag, and Deletion should still work. 

## Changelog text for these changes

Bug: Disallow group deletion after fields have been moved in or out of the group #6937 @zrothauser 

## PR checklist
